src/index.html

```diff
@@ -15,8 +15,92 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a
+              href="#apple"
+              class="is-active nav__link color__words"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="#laptops&computers"
```

Comment:
The 'href' attribute value '#laptops&computers' contains an ampersand ('&'). In HTML, this character should be encoded as '&amp;' to ensure proper rendering and functionality. Consider updating it to '#laptops&amp;computers'.